client/dist/styles.css

```diff
@@ -0,0 +1,79 @@
+
+.app {
+  background-color: #F6F7F8;
+  float: left;
+  padding: 30px;
+}
+
+
+
+.attendees {
+  background-color: #F6F7F8;
+  font: "Graphik Meetup";
+}
+
+.attendee {
+  border-radius: 10px;
+  background-color: white;
+  font: "Graphik Meetup";
+  margin: 1px;
+  text-align: center;
+  padding: 5px 0px 0px 5px;
+  height: 150px;
+  width: 150px;
+}
+
+.attendee_picture {
+  margin-top: 10px;
+  border-radius: 100px;
+  height: 60%;
+  width: 60%;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+
+.attendees_header {
+  background-color: #F6F7F8;
+  display: inline-block;
+  padding-bottom: 20px;
+}
+
+.attendees_count {
+  display: inline-block;
+  font-family: Graphik Meetup,-apple-system,BlinkMacSystemFont,Roboto,Helvetica,Arial,sans-serif;
+  font-size: 20px;
+  text-decoration: bold;
+    background-color: #F6F7F8;
+}
+
+.seeAll {
+  display: inline-block;
+  background-color: #F6F7F8;
+  position: relative;
+  left: 450px;
+  color: #00a2c7;
+  font-family: Graphik Meetup,-apple-system,BlinkMacSystemFont,Roboto,Helvetica,Arial,sans-serif;
+  font-size: 16px;
+}
```

Comment:
Using relative positioning like this will probably break if you resize the module for a bigger or smaller browser.  To get the header and the seeAll to position on opposite sides I'd suggest using flex.  You'd want to add display: flex and justify-content: space-between to the attendees_header.

Reply:
Or you could use floats, but I'd recommend flex.